nixos/modules/services/monitoring/netdata.nix

```diff
@@ -130,6 +141,26 @@ in {
         '';
       };
 
+      configDir = mkOption {
+        type = types.attrsOf types.path;
+        default = {};
+        description = ''
+          Complete netdata config directory except netdata.conf.
+          The default configuration is copied and merged with changes
+          defined in this option.
+          Each top-level attribute denotes a path in the configuration
+          directory as in environment.etc.
+          Its value is the absolute path and must be readable by netdata.
+          Cannot be combined with configText.
+        '';
+        example = literalExpression ''
+          "health_alarm_notify.conf" = pkgs.writeText "health_alarm_notify.conf" '''
+            sendmail="/path/to/sendmail"
+          ''';
+          "health.d" = "/run/secrets/netdata/health.d";
```

Comment:
Example will not work correctly. The folder `health.d` is created inside a subfolder `health.d`
Result:
```
l /nix/store/w3kv04ycchzz3r03ws1daf8knpfnca0n-netdata-config-d | grep health
lrwxrwxrwx    1 root root     68 янв  1  1970 health_alarm_notify.conf -> /nix/store/54m6scjd3hf1hqpcdym6fww8k6126g96-health_alarm_notify.conf
dr-xr-xr-x    2 root root   4,0K янв  1  1970 health.d
-r--r--r--    1 root root     57 янв  1  1970 health_email_recipients.conf

l /nix/store/w3kv04ycchzz3r03ws1daf8knpfnca0n-netdata-config-d/health.d | grep health
lrwxrwxrwx 1 root root   29 янв  1  1970 health.d -> /run/secrets/netdata/health.d
```
The intended result:
```
l /nix/store/5162yws7lnl3f9nsw5h9ixgafhr764zc-netdata-config-d | grep health
lrwxrwxrwx    1 root root     68 янв  1  1970 health_alarm_notify.conf -> /nix/store/54m6scjd3hf1hqpcdym6fww8k6126g96-health_alarm_notify.conf
lrwxrwxrwx    1 root root     29 янв  1  1970 health.d -> /run/secrets/netdata/health.d
-r--r--r--    1 root root     57 янв  1  1970 health_email_recipients.conf

```

Reply:
Fixed, thanks!